docs-devsite/firestore_.md

```diff
@@ -1090,6 +1128,70 @@ export declare function startAt(...fieldValues: unknown[]): QueryStartAtConstrai
 
 A [QueryStartAtConstraint](./firestore_.querystartatconstraint.md#querystartatconstraint_class) to pass to `query()`<!-- -->.
 
+## deleteAllPersistentCacheIndexes()
+
+Removes all persistent cache indexes.
+
+Please note this function will also deletes indexes generated by `setIndexConfiguration()`<!-- -->, which is deprecated.
+
+<b>Signature:</b>
+
+```typescript
+export declare function deleteAllPersistentCacheIndexes(indexManager: PersistentCacheIndexManager): void;
+```
+
+### Parameters
+
+|  Parameter | Type | Description |
+|  --- | --- | --- |
+|  indexManager | [PersistentCacheIndexManager](./firestore_.persistentcacheindexmanager.md#persistentcacheindexmanager_class) |  |
+
+<b>Returns:</b>
+
+void
+
+## disablePersistentCacheIndexAutoCreation()
+
+Stops creating persistent cache indexes automatically for local query execution. The indexes which have been created by calling `enablePersistentCacheIndexAutoCreation()` still take effect.
+
+<b>Signature:</b>
+
+```typescript
+export declare function disablePersistentCacheIndexAutoCreation(indexManager: PersistentCacheIndexManager): void;
+```
+
+### Parameters
+
+|  Parameter | Type | Description |
+|  --- | --- | --- |
+|  indexManager | [PersistentCacheIndexManager](./firestore_.persistentcacheindexmanager.md#persistentcacheindexmanager_class) |  |
+
+<b>Returns:</b>
+
+void
+
+## enablePersistentCacheIndexAutoCreation()
+
+Enables SDK to create persistent cache indexes automatically for local query execution when SDK believes cache indexes can help improves performance.
```

Comment:
Edit:

"Enables the SDK to create persistent cache indexes automatically for local query execution when the SDK believes cache indexes can help improves performance."